providers/stores/mongo.js

```diff
@@ -177,6 +179,10 @@ class MongoStore {
     if (parameters.license) filter['licensed.declared'] = parameters.license
     if (parameters.releasedAfter) filter['described.releaseDate'] = { $gt: parameters.releasedAfter }
     if (parameters.releasedBefore) filter['described.releaseDate'] = { $lt: parameters.releasedBefore }
+    if (parameters.minEffectiveScore) filter['scores.effective'] = { $gt: parseInt(parameters.minEffectiveScore) }
+    if (parameters.maxEffectiveScore) filter['scores.effective'] = { $lt: parseInt(parameters.maxEffectiveScore) }
+    if (parameters.minToolScore) filter['scores.tool'] = { $gt: parseInt(parameters.minToolScore) }
+    if (parameters.maxToolScore) filter['scores.tool'] = { $lt: parseInt(parameters.maxToolScore) }
```

Comment:
With these new params in place we can now query the definitions for effective and tool score, which would cover https://github.com/clearlydefined/website/issues/501 requirement.
Should we need to add another combined field to query? Like `scores.total` perhaps , which is, actually, the real score of the entire definition, as required in https://github.com/clearlydefined/website/issues/495

Reply:
the real score of the entire definition will be `definition.scores.effective`